include/ginkgo/core/log/profiler_hook.hpp

```diff
@@ -276,12 +277,12 @@ class ProfilerHook : public Logger {
         /** The name of the range. */
         std::string name;
         /** The total runtime of all invocations of the range in nanoseconds. */
-        int64 inclusive_ns{};
+        std::chrono::nanoseconds inclusive{};
```

Comment:
If it is not merged before this release, it will be an interface break.
Maybe you already answered it before, could summary_entry be private?

Reply:
great catch! If necessary, I can pull out just the `nanoseconds` change